### ISSUE TITLE
Oceananigans "wiki" for hosting practical information about using Oceananigans

### ISSUE THREAD

**Issue by glwagner** [CliMA/Oceananigans.jl#2470]
I opened a wiki for Oceananigans:

https://github.com/CliMA/Oceananigans.jl/wiki

I think we should use the wiki to host practical information and tips for using Oceananigans on various hardware (laptops, CPUs, GPUs), clusters / high-performance computing system (HPCs), and more.

Using the wiki this way will allow us to focus the docs on code and numerics, and alleviate the need to submit PRs to update details about using clusters.

We need to provide an introduction to Oceananigans "knowledge base" (where to find useful information in the documentation, wiki, Github Discussions, and Github Issues) in the README, the docs, and the wiki.

**Comment by glwagner** [comment]
Note that we have the option of opening up wiki editing to anyone on GitHub: 

https://docs.github.com/en/communities/documenting-your-project-with-wikis/about-wikis

so we should discuss that.

**Comment by francispoulin** [comment]
Would we prefer to have anyone make a change to the wiki or perhaps post an issue and suggest that we add something to the wiki? 

I am find with either but might have a slight preference for the latter.

**Comment by navidcy** [comment]
Who can edit? I guess those with repo write access, right?

**Comment by glwagner** [comment]
Yeah. I think we should let chaos reign for a little bit. Perhaps for big changes, we can open issues to discuss first.
